pkg/importer/dataStream_test.go

```diff
@@ -14,10 +14,10 @@ import (
 	"testing"
 
 	"kubevirt.io/containerized-data-importer/pkg/image"
-	"kubevirt.io/containerized-data-importer/test/framework"
+	"kubevirt.io/containerized-data-importer/tests/framework"
```

Comment:
I know you know: but this file contains the new `framework` pkg. Is is still relevant to the renaming and restructuring of unit tests?

Reply:
It shouldn't be.  Git will resolve the diffs between these files.  It won't have an effect on unit tests that importer framework.

Reply:
This is probably ok. My concern is that the Framework struct and methods today are motivated and evolving based on e2e usage. That _may_ conflict with unit test usage down the road.

Reply:
@jeffvance The changes here do not affect the framework struct at all.  That said, maybe this 
belongs in a separate package.  `tests/utils/`?